optlib/terraform.ctags

```diff
@@ -0,0 +1,28 @@
+#
+# terraform.ctags: regex parse for Terraform (HCL).
+#
+# Author: Antony Southworth <https://github.com/antonysouthworth-halter>
+#
+# This source code is released for free distribution under the terms of the
+# GNU General Public License version 2 or (at your option) any later version.
+#
+# Derived from `vim-terraform-completion`:
+#   - https://github.com/juliosueiras/vim-terraform-completion/blob/master/ctags/terraform.ctags
```

Comment:
If the regex patterns from @juliosueiras's work, ask @juliosueiras to allow to use the .ctags in GPLv2 or later.

Reply:
what would I need to paste?

Reply:
Could you fill <YOUR NAME HERE> and paste it at the head of your .ctags?
```
#
#  Copyright (c) 2021, <YOUR NAME HERE>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License
# as published by the Free Software Foundation; either version 2
# of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,
# USA.
#
```

Reply:
done

Reply:
Thank you!